src/App.jsx

```diff
@@ -14,70 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam is selected');
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```
You shouldn't store the whole text in state. Just save the product
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/e70e44dc-0d63-4c8e-b991-2141d4359381)